client/src/components/Forums/ThreadDetails/ThreadDetails.tsx

```diff
@@ -41,17 +43,30 @@ const ThreadDetail: React.FC<ThreadDetailProps> = ({ forumId, threadId }) => {
     setCreatingPost(!creatingPost);
   };
 
+  const handleDeletePost = async (postId: string) => {
+    try {
```

Comment:
Please incorporate logic for pending / loading with async functions.

Reply:
Great call @JimmyTran48 . I built in some quick pending states for the delete function but this will be handled with RTK once implemented. There is no actual user feedback for fails right now. There are several Jira tickets up to tackle this.